hoomd/hpmc/ExternalFieldHarmonic.h

```diff
@@ -306,11 +306,12 @@ template<class Shape> class ExternalFieldHarmonic : public ExternalFieldMono<Sha
         return dE;
         }
 
-    /** This function gets called during the update step in the MC integrator
+    /** This function gets exported to python to act as a property
```

Comment:
Except that this isn't exported as a property with pybind. Typically the first sentence in a documentation comment describes what the method does: "Compute the total external energy on the system."